IoTConnect_2.0/IoTConnect_2.0/IoTConnectManager/IoTConnectManager.swift

```diff
@@ -368,10 +408,14 @@ class IoTConnectManager {
                 if dictD?.isEmpty == true{
                     if let valDict = value as? [String:Any]{
                         for (valDictKey,valDictValue) in valDict{
-                            dictD?.append(anotherDict: [    "\(key)":[valDictKey:["\(valDictValue)","\(valDictValue)","\(valDictValue)","\(valDictValue)","1.0","\(valDictValue)"]]])
+//<<<<<<< Updated upstream
```

Comment:
remove unwanted code

Reply:
@sw-ot-deveshmevada Changes are done in new push